index.html

```diff
@@ -201,5 +201,19 @@ <h3>Skills</h3>
       </div>
   </section>
   </main>
+  <footer>
+    <div class="bottom">
+        <h2 class="white">Contact me </h2>
+        <p class="call">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/xkneznwb"
+        method="POST">
+
+            <input type="text" id="fname" name="name" placeholder="Your Name" value="Alvaro Iporre Martinez" maxlength="30" required><br>
+            <input type="email" id="lname" name="email" placeholder="Your Email" required value="alvaroiporremartinez@gmail.com" ><br>
```

Comment:
- [Optional] I suggest removing the value attribute for all your input fields. This will be better for user experience. Currently, the input fields are prefilled, and the user must clear them before they can add their data.